acceptancetests/deploy_stack.py

```diff
@@ -1049,7 +1049,11 @@ def runtime_context(self, addable_machines):
                 with self.client.ignore_soft_deadline():
                     self.client.list_controllers()
                     self.client.list_models()
-                    for m_client in self.client.iter_model_clients():
+
+                    # Only show status for models the backend is tracking.
+                    # This prevents errors attempting to retrieve status for
+                    # models that have been issued a destroy command.
+                    for m_client in self.client._backend.added_models:
```

Comment:
Of course, we shouldn't really be accessing this protected property, but we already do it in multiple places, so it's a hill not worth dying on.